tensorflow/python/tools/import_pb_to_tensorboard.py

```diff
@@ -41,19 +41,19 @@
 
 
 def import_to_tensorboard(model_dir, log_dir, tag_set):
-  """View an imported protobuf model (`.pb` file) as a graph in Tensorboard.
+  """View an SavedModel as a graph in Tensorboard.
 
   Args:
-    model_dir: The location of the protobuf (`pb`) model to visualize
+    model_dir: The directory containing the SavedModel to import.
```

Comment:
directory containing the imported/saved protobuf (`pb`) model to visualize.

Reply:
ditto